specs/DotNetProjectFile.Analyzers.Specs/Rules/Generic/Remove_commented_out_code.cs

```diff
@@ -0,0 +1,47 @@
+namespace Rules.Generic.Remove_commented_out_code;
+
+public class Reports
+{
+    [Test]
+    public void on_XML_commented_out_in_MS_Build() => new RemoveCommentedOutCode()
+       .ForInlineCsproj(@"
+<Project Sdk=""Microsoft.NET.Sdk"">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <!-- ImplicitUsings>enable</ImplicitUsings -->
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Reconsider adding this
+    <GlobalPackageReference Include=""DotNetProjectFile.Analyzers"" Version=""1.5.8"" />
+    -->
+  </ItemGroup>
+
+</Project>")
+       .HasIssues(
+           Issue.WRN("Proj3002", "Remove the commented-out code").WithSpan(04, 08, 04, 47),
+           Issue.WRN("Proj3002", "Remove the commented-out code").WithSpan(08, 08, 10, 04));
+
+    [Test]
+    public void on_XML_commented_out_in_RESX() => new Resx.RemoveCommentedOutCode()
+       .ForProject("ResxCommentedOut.cs")
+       .HasIssues(
+           Issue.WRN("Proj3002", "Remove the commented-out code").WithSpan(17, 06, 21, 02).WithPath("Resources.resx"),
+           Issue.WRN("Proj3002", "Remove the commented-out code").WithSpan(25, 06, 25, 19).WithPath("Resources.resx"));
+}
+
+public class Guards
```

Comment:
Could you maybe add a case with a proper comment and a case with a todo to ensure they don't cause this analyzer to report?

Reply:
Done.